src/libs/actions/IOU.js

```diff
@@ -588,6 +601,11 @@ function cancelMoneyRequest(chatReportID, iouReportID, type, moneyRequestAction)
             key: `${ONYXKEYS.COLLECTION.REPORT}${iouReportID}`,
             value: updatedIOUReport,
         },
+        {
+            onyxMethod: CONST.ONYX.METHOD.SET,
+            key: `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`,
+            value: null,
```

Comment:
Wondering if the transaction detail view should be pending too, ie if you could still load the page and it would be disabled essentially.

Thats probably too complicated though

Reply:
You ill have a transaction details page with $10 request for example and you will delete this transaction. Now the detailed page (`/r/:reportID/t/:transactionID`) can still be accessed as the thread, but how will we display it? the transaction data is null so we cannot show anything in the description or amount field.

I assume we will only have to show the original thread